<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cinema</class>
 <widget class="QMainWindow" name="Cinema">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cinema</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="toolButtons" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <widget class="QPushButton" name="homeButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>42</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="pages">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="actorsPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="actorsLabel">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Actors</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="actorsScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="actorsScrollAreaWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>620</width>
             <height>358</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sessionsPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="labelSessions">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Sessions</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="1" column="1" colspan="2">
         <widget class="QPushButton" name="sessionsButton">
          <property name="text">
           <string>Сеансы</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QPushButton" name="todaySessionsButton">
          <property name="text">
           <string>Сеансы сегодня</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QPushButton" name="directorsButton">
          <property name="text">
           <string>Режиссеры</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QPushButton" name="filmsButton">
          <property name="text">
           <string>Фильмы</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QPushButton" name="actorsButton">
          <property name="text">
           <string>Актеры</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5" colspan="2">
         <widget class="QPushButton" name="hallsButton">
          <property name="text">
           <string>Залы</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5" colspan="2">
         <widget class="QPushButton" name="clientsButton">
          <property name="text">
           <string>Клиенты</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" colspan="3">
         <widget class="QPushButton" name="todayFilmsButton">
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Фильмы сегодня</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="filmsPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="filmsLabel">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Films</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="filmsScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="filmsScrollAreaWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>66</width>
             <height>16</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
